generators/entity/templates/server/src/main/java/package/service/mapper/_EntityMapper.java

```diff
@@ -47,9 +47,10 @@
     List<<%= entityClass %>> <%= entityInstance %>DTOsTo<%= entityClassPlural %>(List<<%= entityClass %>DTO> <%= entityInstance %>DTOs);
 
     /**
-     * generating the fromId for all mappers, as the class has relationship to it might need it, instead of
+     * generating the fromId for all mappers if the databaseType is sql, as the class has relationship to it might need it, instead of
      * creating a new attribute to know if the enitity has any relationship from someother entity
```

Comment:
Spelling `enitity` -> `entity` and `someother` -> `some other`

I'll test this today